app/templates/admin/cceMinor.html

```diff
@@ -45,21 +45,31 @@ <h2>CCE Minor Progress</h2>
     </tbody>
 </table>
 <br>
-<h2>Interested Students</h2>
+<div class="row">
+    <div class="col">
+        <h2>Interested Students</h2>
+    </div>
+    <div class="col-auto">
+        <button class="btn btn-primary" id="emailAllInterested">Email All</button>
+    </div>
+</div>
 <table class="display" id="interestedStudentsTable">
     <thead>
         <tr>
             <th>Name</th>
         </tr>
     </thead>
     <tbody>
+        {% set interestedStudentEmails = [] %}
         {% for student in interestedStudentsList %}
+            {% set _ = interestedStudentEmails.append(student.email) %}
```

Comment:
If there is a possibility to handle that in the backend before rendering, you should go for it. Having this operation done in the route would be awesome, given that jinja should be the last option for this kind of operation.

Reply:
Good idea. This has been done